XenModel/Messages.resx

```diff
@@ -1874,12 +1874,18 @@ Archive destination: {0}</value>
   <data name="ASSIGNED_PROTECTION_POLICY" xml:space="preserve">
     <value>Protection policy assigned.</value>
   </data>
+  <data name="ASSIGNED_VMSS_POLICY" xml:space="preserve">
+    <value>Snapshot schedule policy assigned.</value>

```

Comment:
The phrase "snapshot schedule policy" sounds clumsy to me. I would rather just call it a "snapshot schedule", i.e. without the "policy". (This affects lots of strings).